app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Dish: <%= @dish.name %></h1>
+
+<h2>Description: <%= @dish.description %>
+
+<div id="ingredients">
+  <h3>Ingredients</h3>
+  <% @dish.ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
```

Comment:
This is where you could add a button to delete the ingredient from the dish, and that could also route to the DishIngredientsController